### ISSUE TITLE
Add 3-augment from DeiT III

### ISSUE THREAD

**Issue by trawler0** [pytorch/vision#8514]
### 🚀 The feature

As the title suggest, add the data augmentation from https://arxiv.org/abs/2204.07118

### Motivation, pitch

This seems to be a simple recipe with good results and the Deit family is widely recognized.

**Comment by Bhavay-2001** [comment]
Hi @trawler0, did you mean something like [this](https://github.com/facebookresearch/deit/blob/main/augment.py)?

**Comment by trawler0** [comment]
Yes, that's what I mean, I should have maybe directly attached the file.
The recipe helped them to train very large ViT models from scratch on imagenet and they got amazing results.

**Comment by Bhavay-2001** [comment]
Hi @NicolasHug, any views on this? Should I try to add this augmentation to torchvision?

**Comment by NicolasHug** [comment]
Hi @trawler0 , thank you for feature request. Sure, I think this can be in scope of our augmentation strategies, even if ultimately the implementation will just be a `Compose` of a few building blocks.

@Bhavay-2001 thanks for offering your help. Let's see if @trawler0 would like to give this a go first, and if not then I'm happy for you to get to it.
Thanks!

**Comment by trawler0** [comment]
Hi @NicolasHug I am happy to add this feature.
